tests/compiler/test_execution.py

```diff
@@ -39,9 +39,6 @@ def test_issue_139_scheduler(snippetcompiler):
 
 f = std::testing::NullResource(name=host.name)
 
-std::Service(host=host, name="svc", state="running", onboot=true, requires=[f])
-ref = std::Service[host=host, name="svc"]
-
```

Comment:
This is the entire purpose of the test, you have to replace it with some other index lookup

Reply:
Am I missing something? This is on the old version (with the std::Service and std==5.2.4):
```
        else:
>           raise MultiException(excns)
E           inmanta.ast.MultiException: Reported 2 errors:
E           	The object __config__::Host (instantiated at /tmp/tmpdb_a7l7j/main.cf:10) is not complete: attribute attr (/tmp/tmpdb_a7l7j/main.cf:6:12) is not set
E           	The object __config__::Host (instantiated at /tmp/tmpdb_a7l7j/main.cf:10) is not complete: attribute host_config (/home/hugo/Documents/Inmanta/copies/inmanta-core/venv/lib/python3.11/site-packages/inmanta_plugins/std/model/_init.cf:463:6) is not set
```
This is on the new one (std==6.0.0):
```
>           raise MultiException(excns)
E           inmanta.ast.MultiException: Reported 2 errors:
E           	The object __config__::Host (instantiated at /tmp/tmpcj4mkegw/main.cf:10) is not complete: attribute attr (/tmp/tmpcj4mkegw/main.cf:6:12) is not set
E           	The object __config__::Host (instantiated at /tmp/tmpcj4mkegw/main.cf:10) is not complete: attribute host_config (/home/hugo/Documents/Inmanta/copies/inmanta-core/venv/lib/python3.11/site-packages/inmanta_plugins/std/model/_init.cf:286:6) is not set
```

Reply:
yes, which are both the good behavior. But the testcase no longer reproduces the original bug, so the fact that it doesn't exhibit the bad behavior doesn't indicate it is absent. 

(the annoying thing is that it is now very hard to reproduce the bug, because it has been fixed a long time ago, but we want to keep this regression test in here)